### ISSUE TITLE
tpm2_createpolicy segfaults if more than 12 PCRs are specified

### ISSUE THREAD

**Issue by oberien** [tpm2-software/tpm2-tools#1093]
When executing the following command with tpm2-tools 3.0.4
```sh
sudo tpm2_createpolicy -T device -f policy_pcr.file -P -g sha1 -L sha1:0,1,2,3,4,5,6,7,8,9,10,11,12
```
if more than 12 registers are specified (like 13 in this example), the function `Tss2_Sys_GetTctiContext` segfaults in the function `sapi_teardown_full`. The passed `sapi_context` pointer points to nonexistent memory according to gdb.

The policy is sucessfully written to the file, because it only segfaults in the teardown after command execution is finished.

https://github.com/tpm2-software/tpm2-tools/blob/12bfda88f02fa056ac10fbe8f1f901afa9b0ee67/tools/tpm2_tool.c#L59-L69

**Comment by williamcroberts** [comment]
Appears fixed on 9381559b055b4b7:
```
$ tpm2_pcrlist -L sha1:0,1,2,3,4,5,6,7,8,9,10,11,12 -o pcrs.out
$ tpm2_createpolicy --policy-pcr -F pcrs.out -g sha1 -L sha1:0,1,2,3,4,5,6,7,8,9,10,11,12 -o policy.dat
ERROR: Number of PCR is limited to 8
ERROR: Could not build pcr policy
ERROR: Unable to run tpm2_createpolicy
```

**Comment by williamcroberts** [comment]
Duplicate of #778
